frontend/src/scenes/billing/BillingLimitInput.tsx

```diff
@@ -13,87 +12,106 @@ import { billingProductLogic } from './billingProductLogic'
 export const BillingLimitInput = ({ product }: { product: BillingProductV2Type }): JSX.Element | null => {
     const limitInputRef = useRef<HTMLInputElement | null>(null)
     const { billing, billingLoading } = useValues(billingLogic)
-    const { isEditingBillingLimit, showBillingLimitInput, customLimitUsd } = useValues(
+    const { isEditingBillingLimit, customLimitUsd } = useValues(
         billingProductLogic({ product, billingLimitInputRef: limitInputRef })
     )
     const { setIsEditingBillingLimit, setBillingLimitInput, submitBillingLimitInput } = useActions(
         billingProductLogic({ product })
     )
 
-    if (!showBillingLimitInput || !product.subscribed) {
+    if (billing?.billing_period?.interval !== 'month' || !product.subscribed) {
         return null
     }
+
     return (
         <Form formKey="billingLimitInput" props={{ product: product }} logic={billingProductLogic} enableFormOnSubmit>
             <div className="border-t border-border p-8" data-attr={`billing-limit-input-${product.type}`}>
+                <h4 className="my-4">Billing limits</h4>
```

Comment:
Recommend bumping to h3, removing the my-4 

<img width="924" alt="Screenshot 2024-06-05 at 2 22 02 PM" src="https://github.com/PostHog/posthog/assets/21014901/8febc8f0-1282-4016-b81b-47e9b766d662">

Reply:
I think it's weird to say `/ billing_period` (which is pretty much always month? I thought even annual customers reset usage on a monthly basis w/ the new credit system) here. This billing limit doesn't change on a month to month basis, the usage counter gets resets on a month to month basis. I think we can do a better job explaining that their usage gets reset every billing cycle somewhere else on the page.

Reply:
I kind of liked how the old UI had the dollar amount highlighted orange to indicate that it was editable. I don't know if that's too subtle, but I don't think anybody had issues figuring it out once we told them that they could find the button to enable the limits by clicking the three dot hamburger menu. 

I also debated using the <IconPencil /> to indicate editable...maybe we make the dollar amount orange and put the pencil icon next to it?

Reply:
> Recommend bumping to h3, removing the my-4

I'm following the patterns of the above tile but I can take a look. 

> I think it's weird to say / billing_period

I'm not sure I understand. I think it's good to specific the period the billing limit is for.